gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -665,6 +666,16 @@ func (vm *VMKeeper) Run(ctx sdk.Context, msg MsgRun) (res string, err error) {
 	return res, nil
 }
 
+// QueryNamespacePaths returns public facing function signatures.
+func (vm *VMKeeper) QueryNamespacePaths(ctx sdk.Context, namespace string) []string {
+	domain := vm.getChainDomainParam(ctx)
```

Comment:
With IBC, we’ll be able to import packages from other domains as well. I don’t think we should enforce it; ust expect users to explicitly include the domain in the package prefix when needed.

Reply:
After internal discussion, automatic domain context will be applied only to the username `@` c.f [PR body](https://github.com/gnolang/gno/pull/2969#issue-2594956507)